src/parser/youtube/ItemMenu.ts

```diff
@@ -0,0 +1,59 @@
+import { ParsedResponse } from '..';
+import Actions from '../../core/Actions';
+import { InnertubeError } from '../../utils/Utils';
+import Button from '../classes/Button';
+import Menu from '../classes/menus/Menu';
+import MenuServiceItem from '../classes/menus/MenuServiceItem';
+import NavigationEndpoint from '../classes/NavigationEndpoint';
+
+class ItemMenu {
+  #page: ParsedResponse;
+  #actions: Actions;
+  #menu: Menu;
+  constructor(data: ParsedResponse, actions: Actions) {
+    this.#page = data;
+    this.#actions = actions;
+    const menu = data?.live_chat_item_context_menu_supported_renderers;
+    if (!menu || !menu.is(Menu)) {
+      throw new InnertubeError('Response did not have an "live_chat_item_context_menu_supported_renderers" property. The call may have failed.');
+    }
+    this.#menu = menu.as(Menu);
```

Comment:
How about we expose only the `items` prop of the menu in this class? I think it would make more sense since it's already named `ItemMenu`. That way, instead of having to rely on the `getter` (e.g; `itemMenu.menu().items`), they could be accessed through `itemMenu.items` which is a cleaner in my opinion.

Reply:
Yours is better. I fixed it.